projects/Mallard/src/components/front/card-group/card.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 import { View, StyleSheet, StyleProp, ViewStyle } from 'react-native'
-import { metrics } from '../../theme/spacing'
+import { metrics } from '../../../theme/spacing'
```

Comment:
I think we should try and figure out project absolute paths as these long imports are getting in the way.

Reply:
seems pretty straightforward, agreed we should do it https://medium.com/@davidjwoody/how-to-use-absolute-paths-in-react-native-6b06ae3f65d1